README.md

```diff
@@ -1,2 +1,28 @@
-# reservations
-Module for booking reservations
+# rent2nite reservations system
+
+This repository serves as the reservation system for the rent2nite application. The repository contains an interactive calendar that allows a user to book a reservation if the chosen timeslot is available.
+
```

Comment:
Excellent description of module functionality.  Cannot wait to see this fleshed out